src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  let amount = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    amount -= LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
+    amount -= SHORT_TERM_DISCOUNT;
+  }
```

Comment:
Instead of using `else if` after an `if` with a `return`, you can simplify the control flow by using just `if` statements. After the discount is applied and the amount is returned, the function execution will stop, making the `else` unnecessary. This will improve the readability and reduce the nesting of your code.